packages/server/src/api/routes/tests/backup.spec.ts

```diff
@@ -19,11 +19,8 @@ describe("/backups", () => {
 
   describe("/api/backups/export", () => {
     it("should be able to export app", async () => {
-      const { body, headers } = await config.api.backup.exportBasicBackup(
-        config.getAppId()!
-      )
+      const body = await config.api.backup.exportBasicBackup(config.getAppId()!)
       expect(body instanceof Buffer).toBe(true)
-      expect(headers["content-type"]).toEqual("application/gzip")
```

Comment:
This check might be important. We should probably keep it

Reply:
It's still happening, just lower down the stack. It's in `BackupAPI`, because all callers of this endpoint should assert this type on success.